### DISCUSSION TITLE
Daily Standup (Monday, June 13th)

### DISCUSSION THREAD

**Discussion by BrandonMagana** [CarlosCRG19/ijalti-peers#126] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:

```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 -  I have been mainly working on storing the images on the db

**What are your priorities for the next 24 hours?**:
 -  Figure out how to store that data

**Blockers**:
 - There's not many info about this

**Shoutouts**:
 - @CarlosCRG19 @antonieto @DavBot02  for their first day of internship

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
 - Finished Aspirant Feed

**What are your priorities for the next 24 hours?**:
 - Solve details from aspirant feed
 - Implement proper routing for aspirant search and aspirant feed
 - Take another task

**Blockers**:
 - None. All good

**Shoutouts**:
 - @everyone
